cmd/telemetrygen/internal/logs/config.go

```diff
@@ -24,4 +25,5 @@ func (c *Config) Flags(fs *pflag.FlagSet) {
 
 	fs.IntVar(&c.NumLogs, "logs", 1, "Number of logs to generate in each worker (ignored if duration is provided)")
 	fs.StringVar(&c.Body, "body", "the message", "Body of the log")
+	fs.StringVar(&c.SeverityText, "severity", "Info", "Severity of the log, one of: Trace, Debug, Info, Warn, Error, Fatal")
```

Comment:
Since we have `SeverityText` and `SeverityNumber`, I think we should call this `severity-text`